File: docs/components_page/components/nav/fill.R
library(dashBootstrapComponents)

nav_contents <- list(
  dbcNavItem(dbcNavLink("Active", href = "#", active = TRUE)),
  dbcNavItem(dbcNavLink("A much longer link label", href = "#")),
  dbcNavItem(dbcNavLink("Link", href = "#"))
)

nav1 <- dbcNav(nav_contents, pills = TRUE, fill = TRUE)

nav2 <- dbcNav(nav_contents, pills = TRUE, justified = TRUE)

navs <- div(list(nav1, html$hr(), nav2))

File: docs/components_page/components/nav/fill.jl
using DashBootstrapComponents

nav_contents = [
    dbc_navitem(dbc_navlink("Active", href = "#", active = true)),
    dbc_navitem(dbc_navlink("A much longer link label", href = "#")),
    dbc_navitem(dbc_navlink("Link", href = "#")),
]

nav1 = dbc_nav(nav_contents, pills = true, fill = true);

nav2 = dbc_nav(nav_contents, pills = true, justified = true);

navs = html_div([nav1, html_hr(), nav2]);
